README.md

```diff
@@ -17,3 +18,32 @@ Happraisal is an application that makes people working on the appraisal **h**app
 - [] Hexagonal Architecture with ArchUnit
 - [] Mutation tests (pitest)
 
+
+
+# Tests
+* [x] Create unit-test.gradle file
+    * [x] add dependencies 
+
+* [ ] Create integration-test.gradle file
```

Comment:
We should consider this task completed (and mark it as done).

Reply:
@dcremonini done